components/ButtonLink.module.scss

```
@@ -0,0 +1,37 @@
+%button-base {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 5.4rem;
+  font-size: 1.8rem;
+  font-weight: 600;
+  color: var(--white-color);
+  text-align: center;
+  cursor: pointer;
+  border-radius: 0.8rem;
+  transition: opacity 300ms ease-in-out;
+
+  &:disabled {
+    cursor: not-allowed;
+    background-color: rgba(0, 0, 0, 0.2);
+  }
+
+  &:hover {
```

hover와 같은 css를 작성하실때 disabled 상태에서도 hover 관련 css가 동작하는지 확인해보시면 좋을것 같습니다.
지금 작성하신 css로는 disabled 상태에서도 hover가 가능할 것으로 보여집니다.
이런 경우에는 아래와 같이 작성을 제안드릴 수 있을것 같습니다.

```suggestion
  &:enabled:hover {
```